src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -123,7 +126,7 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
 
     private fun checkUnsafeCall(holder: AnnotationHolder, o: RsExpr) {
         if (!o.isInUnsafeBlockOrFn(/* skip the expression itself*/ 1)) {
-            val annotation = holder.createErrorAnnotation(o, "Call to unsafe function requires unsafe function or block [E0133]")
+            val annotation = RsDiagnostic.UnsafeError(o, "Call to unsafe function requires unsafe function or block").addToHolder(holder)
             annotation.registerFix(SurroundWithUnsafeFix(o))
```

Comment:
On the idea, all the fixes should be added inside a `addToHolder` method, and so it shouldn't return an `Annotation`